src/containers/CheckoutPage/CheckoutPage.js

```diff
@@ -539,8 +541,8 @@ CheckoutPageComponent.propTypes = {
   }).isRequired,
   sendOrderRequest: func.isRequired,
   onCreateStripePaymentToken: func.isRequired,
-  stripePaymentTokenInProgress: bool.isRequired,
-  stripePaymentTokenError: bool.isRequired,
+  stripePaymentTokenInProgress: bool,
+  stripePaymentTokenError: bool,
```

Comment:
Use actual error object (and validate propTypes with our own `propTypes.error` from `../../util/types`)